src/components/FormAddTrip.spec.js

```diff
@@ -8,15 +8,15 @@ describe('FormAdd', () => {
       <FormAddTrip />
     </MemoryRouter>
   );
-  it('renders two inputs, a select and two buttons', () => {
+  it('renders four inputs and two buttons', () => {
     const input1 = screen.getByLabelText(/destination/i);
     const input2 = screen.getByLabelText(/location/i);
-    const select = screen.getByLabelText(/status/i);
+    const dateInputs = screen.getAllByLabelText(/date/i);
     const buttons = screen.getAllByRole('button');
```

Comment:
I am not sure but could you not search for all 4 input fields with `.getAllByRole('input')` ?

Reply:
tried that before, unfortunately it didn't work..